libs/template/helpers.go

```diff
@@ -128,8 +128,8 @@ func loadHelpers(ctx context.Context) template.FuncMap {
 				metastore, err := w.Metastores.Current(ctx)
 				if err != nil {
 					var aerr *apierr.APIError
-					if errors.As(err, &aerr) && aerr.ErrorCode == "METASTORE_DOES_NOT_EXIST" {
-						// Workspace doesn't have a metastore assigned, ignore error
+					if errors.As(err, &aerr) && (aerr.ErrorCode == "PERMISSION_DENIED" || aerr.ErrorCode == "METASTORE_DOES_NOT_EXIST") {
```

Comment:
Were you able to repro the error?

The report mentions a custom access denied message and doesn't mention this error code.

Reply:
I'm trying to get more information from the backend team on this. It's unclear when this happens; I don't think it should happen at all. But the source code for this message does show this error code.

Reply:
Ack, please add this to the test section of the PR summary so we capture where it comes from.

Reply:
Updated, PTAL.

Reply:
LGTM, thanks.